op-batcher/batcher/driver.go

```diff
@@ -409,7 +414,9 @@ func (l *BatchSubmitter) sendTransaction(ctx context.Context, txdata txData, que
 		candidate.GasLimit = intrinsicGas
 	}
 
-	queue.Send(txdata, *candidate, receiptsCh)
+	go func() {
+		queue.Send(txdata, *candidate, receiptsCh)
+	}()
```

Comment:
```suggestion
	go queue.Send(txdata, *candidate, receiptsCh)
```

Reply:
with receipt handling in a separate loop we no longer need async here.